src/async-components/views/dialogs/security/CreateKeyBackupDialog.tsx

```diff
@@ -17,96 +17,112 @@ limitations under the License.
 
 import React, { createRef } from 'react';
 import FileSaver from 'file-saver';
-import * as sdk from '../../../../index';
 import { MatrixClientPeg } from '../../../../MatrixClientPeg';
-import PropTypes from 'prop-types';
 import { _t, _td } from '../../../../languageHandler';
 import { accessSecretStorage } from '../../../../SecurityManager';
 import AccessibleButton from "../../../../components/views/elements/AccessibleButton";
 import { copyNode } from "../../../../utils/strings";
 import PassphraseField from "../../../../components/views/auth/PassphraseField";
-
-const PHASE_PASSPHRASE = 0;
-const PHASE_PASSPHRASE_CONFIRM = 1;
-const PHASE_SHOWKEY = 2;
-const PHASE_KEEPITSAFE = 3;
-const PHASE_BACKINGUP = 4;
-const PHASE_DONE = 5;
-const PHASE_OPTOUT_CONFIRM = 6;
+import { IPreparedKeyBackupVersion } from 'matrix-js-sdk/src/crypto/backup';
+import { IValidationResult } from '../../../../components/views/elements/Validation';
+import Field from '../../../../components/views/elements/Field';
+import BaseDialog from "../../../../components/views/dialogs/BaseDialog";
+import DialogButtons from "../../../../components/views/elements/DialogButtons";
+import Spinner from '../../../../components/views/elements/Spinner';
+
+enum CreateKeyBackupPhases {
+    Passphrase = 0,
+    PassphraseConfirm = 1,
+    ShowKey = 2,
+    KeepItSafe = 3,
+    BackingUp = 4,
+    Done = 5,
+    OptOutConfirm = 6,
```

Comment:
You can omit the explicit values here, or you could just leave the `= 0` for an explicit start value then the rest will be auto-incremented. Given that this wasn't exported before it doesn't need any backwards compat